project/dexter-monitor/routes/server.js

```diff
@@ -155,6 +178,7 @@ function createEmailContentsWhenServerDown(server){
 
 function stopCheckingServerStatus(){
     clearInterval(checkingServerStatus);
+    checkingServerStatus = null;
```

Comment:
`checkingServerStatus` has an object which is returned from `setInterval()` when `startCheckingServerStatus()` is called. So I thought that using `null` value is better than the boolean value `false`.